pic32/cores/pic32/USB_CDCACM.cpp

```diff
@@ -281,14 +289,69 @@ size_t CDCACM::write(uint8_t b) {
         if (avail > CDCACM_BULKEP_SIZE) {
             avail = CDCACM_BULKEP_SIZE;
         }
+		
+		if((_txTail + avail) > CDCACM_BUFFER_SIZE)
```

Comment:
Same here?

Reply:
I will test that in the afternoon. 
It can be implemented as `_txTail + avail >= CDCACM_BUFFER_SIZE`

Reply:
Or `> CDCACM_BUFFER_SIZE - 1`... many ways to skin that cat :)